rpcd/playbooks/roles/rpc_pre_upgrade/tasks/kill_beaver.yml

```diff
@@ -0,0 +1,57 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Make sure beaver is stopped
+  service:
+    name: beaver
+    enabled: no
+    state: stopped
+  tags:
+    - kill-the-beaver
+
```

Comment:
Let's please use something a bit more professional sounding for the tag.
I'd also say that the tag on a single task isn't much use - you probably want to tag all these tasks.